README.md

```diff
@@ -1,20 +1,31 @@
 # Setup
 
-1. Make sure you have Python 3.7+ installed
+1. Make sure you have Python 3.7+ and [pipenv](https://github.com/pypa/pipenv) installed
```

Comment:
This should either read `Python 3.8+` or the Pipfile should specify `3.7` instead of `3.8`. I don't really care if we're going to target `3.7` or `3.8`.

Reply:
you are right, I removed this section